scripts/typescript/src/bcdb/bcdb.ts

```diff
@@ -208,26 +208,19 @@ export class BCDBLoader {
             key: "invariant_degree_of_polymerization",
             type: 'value',
             value: row[Column.N],
-            //method: row[Column.N_method], TODO: fix data to match with vocab
+            method: row[Column.N_method],
             condition: [temperature],
             citation,
             unit: null,          
           });
 
-          // can be: "saxs", "tem", or "rheology"
-          let phase_method = row[Column.phase_method]?.toLowerCase();
-
-          // The custom vocabulary does not allow "rheology"
-          // Replacing with "rheometer".
-          if(phase_method === "rheology")  phase_method = "rheometer";
-
```

Comment:
The XLSX now contains valid lower case values.

Reply:
nice! cleaning up code as you got! love it!